showcase/src/app/welcome/pages/news/_news-page.component.scss

```diff
@@ -1,31 +1,93 @@
-.first-news-image {
-  background: url("/assets/images/screenshots/showcase-screenshot.png") no-repeat center center;
-  width: 250px;
-  height: 60px;
-  display: inline-block;
-  background-size: contain;
-}
-
-.second-news-image {
-  height: 60px;
-  width: 200px;
-
-  background-image: url(/assets/stark-core/logo/dark/stark_logo_dark.svg);
-  background-size: contain;
-  background-position: center;
-  background-repeat: no-repeat;
-}
-
-@media #{$desktop-query} {
-  .first-news-image {
-    width: 750px;
-    height: 200px;
+.fourth-news-image {
+	background: url("/assets/images/screenshots/10.0.0-beta3.PNG");
+  }
+
+  .third-news-image {
+	background: url("/assets/images/screenshots/10.0.0-beta2.PNG");
+  }
+
+  .second-news-image {
+	background: url("/assets/images/screenshots/10.0.0-beta1.PNG");
   }
-}
 
-@media #{$tablet-only-query} {
   .first-news-image {
-    width: 500px;
-    height: 100px;
+	background-image: url(/assets/stark-core/logo/dark/stark_logo_dark.svg);
+  }
+
+  .news-container, .top-news {
+	justify-content: center;
+	align-items: center;
+	flex-direction: column;
+	display: flex;
+  }
+
+  .custom-button {
+	align-items: center;
+	justify-content: center;
+	width: 150px;
+	text-decoration: none;
+	height: 30px;
+	font-size: 13px;
+	font-weight: 600;
+	line-height: 30px;
+	color: #fff;
+	background-color: #0063bb;
+	border-radius: 48px;
+	box-shadow: 0 2px 5px 0 rgba(0, 0, 0, .26);
+	box-sizing: border-box;
+	cursor: pointer;
+	display: block;
+	margin: 15px auto auto auto;
+	padding: 0 24px;
+	text-align: center;
+  }
+  
+  @media #{$desktop-lg-query},#{$desktop-query}, #{$tablet-toc-query-screen} {
```

Comment:
Same remark as before: it seems the styles are inverted... so we need to refactor them to have only one (or two) media queries